mitiq/benchmarks/volume_circuits.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (C) 2022 Unitary Fund
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Functions for creating circuits of the form used in quantum
+volume experiments as defined in https://arxiv.org/abs/1811.12926.
```

Comment:
Can you please add this paper to the references, if it is not included yet? 
The references file is [here](https://github.com/unitaryfund/mitiq/blob/master/docs/source/refs.bib).